dashboard/app/controllers/test_controller.rb

```diff
@@ -450,4 +450,15 @@ def accept_parental_request
     Services::ChildAccount.grant_permission_request!(permission_request)
     head :ok
   end
+
+  # Endpoint for testing DCDO mocking.
+  # @see /dashboard/test/ui/features/dcdo_mocking.feature
+  def get_dcdo
+    render json: {
+      fetched: DCDO.get('test_dcdo_on_pegasus', nil),
```

Comment:
can we update this key to be more accurate, now that this is no longer testing DCDO on pegasus?